bincompat-c-hello/scripts/common.sh

```diff
@@ -0,0 +1,17 @@
+ELFLOADER_BASE="elfloader-basic"
+
+if ! test -d workdir; then
+    mkdir workdir
+fi
+
+ln -sfn "../../$ELFLOADER_BASE/workdir" workdir/base
+
+build_elfloader_base () {
+    (
+        set -e
+        cd "../$ELFLOADER_BASE"
+        ./setup.sh
+        make distclean
+        "./scripts/build/$1"
+    ) || exit 1
```

Comment:
This won't work. It will run `exit 1` in a subshell. Use `if test $? -ne 0; then exit 1; fi` if you want to exit the script.